templates/unit.j2

```diff
@@ -34,7 +34,7 @@ ExecStart={{ docker_path }} run \
   {{ params('--device', container_devices) }}\
   {% if container_privileged == true %}--privileged{% endif %}\
   {{ container_args | trim }} \
-  {{ container_image }} {{ container_cmd | default('') | trim }}
+  {{ container_image }} {{ container_cmd | join(' ') }}
```

Comment:
What is the advantage (besides the commets) of using this vs

```
container_cmd: >-
  --entrypoints.websecure.address=:443
  --api
  --providers.docker=true
  --providers.docker.exposedByDefault=false
  --certificatesresolvers.letsencrypt.acme.dnschallenge=true
  --certificatesresolvers.letsencrypt.acme.dnschallenge.provider=digitalocean
  --certificatesResolvers.letsencrypt.acme.tlsChallenge=true
  --certificatesresolvers.letsencrypt.acme.email=e@mail.com
  --certificatesresolvers.letsencrypt.acme.storage=/letsencrypt/acme.json
```

Otherwise, a 

    {% if container_cmd is string %}{{ container_cmd }}{% else %}{{ container_cmd | join(' ') }}{% endif %}

or similar would keep the change backwards compatible

Reply:
Oh, I was not aware of this version with `container_cmd: >-`.
Personally, I see setting comments as a big benefit, especially when you have a long command list. Things like, enabling the staging server for Lets encrypt quickly and related.
Or adding tickets to specific commands, etc.

What do you think about having it as string and/or list?
Is this Ansible native or more discouraged?

Let me know what you think and I will adjust the docs.

Reply:
Yes I agree about the comments! As long as backwards-compatibility is maintained :)